topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "fmt"
+
+func main() {
+	// Run ReverseString
+	fmt.Println(ReverseString("Hello World"))
+}
+
+func ReverseString(str string) string {
+	// Create rune.
+	chars := []rune(str)
```

Comment:
This is creating a slice of runes not an indivudual rune. This is not really chars but a set of code points. So the variable name is not accurate.

Reply:
I was not sure what name would be best here, so I want with **str_table**.